### PR TITLE
make source containers aware of cachito

### PR DESCRIPTION
**Pull request by rcerven**
* OSBS-8233

Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

File: docs/users.rst [hunk 1]
```diff
@@ -1248,8 +1248,10 @@ available in the OSBS buildroot as `bsi` executable in `$PATH`.
 
 
 Current limitations:
-* only Source RPMs are added into source container image
+
+* only Source RPMs and sources fetched through cachito integration are added into source container image
```
Comment (MartinBasti) [inline docs/users.rst]: Is possible to link `cachito` to cachito section?

Comment (rcerven) [reply docs/users.rst]: added

File: docs/users.rst [hunk 2]
```diff
@@ -1248,8 +1248,10 @@ available in the OSBS buildroot as `bsi` executable in `$PATH`.
 
 
 Current limitations:
-* only Source RPMs are added into source container image
+
+* only Source RPMs and sources fetched through :ref:`cachito-integration` are added into source container image
```
Comment (athos-ribeiro) [inline docs/users.rst]: We should substitute the "*" for "-" in both lines here: This is how lists in rst work.

Comment (athos-ribeiro) [reply docs/users.rst]: NVM, maybe it was broken due to the missing line break!

**Pull request merged**